src/Microsoft.Health.Dicom.Client/DicomWebConstants.cs

```diff
@@ -12,6 +12,7 @@ public static class DicomWebConstants
         public const string BasePartitionUriFormat = "/partitions/{0}";
         public const string BaseStudyUriFormat = "/studies/{0}";
         public const string BaseWorkitemUriFormat = "/workitems/{0}";
+        public const string AddWorkitemUriFormat = "/workitems?{0}";
```

Comment:
We will use the base uri format for other operations, this is unique to add/create.